docs/program-setup/bare_metal/bare_metal_api.md

```diff
@@ -1,47 +1,26 @@
 # Bare metal APIs
 
-Mbed OS bare metal supports the following APIs:
+The bare metal profile starts with a restricted list of API, to which you can add what you need. However, bare metal doesn't support all Mbed OS APIs.
 
-<h2 id="analog-i-o">Analog I/O</h2>
+## Initial list of APIs
 
-- AnalogIn - Read the voltage applied to an analog input pin.
-- AnalogOut - Set the voltage of an analog output pin.
+By default, bare metal includes the following APIs:
 
-<h2 id="digital-i-o">Digital I/O</h2>
+## Adding APIs
```

Comment:
This will be covered in the tutorial. This page shoud only list what is supported and the default list of APIs.